### PR TITLE
Added IIS settings to keep Vonk running

### PR DESCRIPTION
**Pull request by mbaltus**

### CODE REVIEW

File: vonk/deployment/reverseProxy/iis.rst [hunk 1]
```diff
@@ -56,11 +62,24 @@ Create Website in IIS
...
   :align: center
 
-3. Edit the application pool to set the **.NET CLR VERSION** to **NO Managed Code**, similar to the picture below (we use  IIS as a reverse proxy, so it isn’t actually executing any .NET code). To edit the application pool, go to the **Application Pools** panel, right-click the website's app pool and select **Basic Settings...** from the popup menu.
+3. Edit the application pool to set the **.NET CLR VERSION** to **NO Managed Code**, similar to the picture below (we use  IIS as a reverse proxy, so it isn’t actually executing any .NET code). To edit the application pool, go to the **Application Pools** panel, right-click the website's app pool and select **Advanced Settings...** from the popup menu.
 
 .. image:: ../../images/iis_edit_application_pool.png
   :align: center
 
+.. attention::
+   IIS will pauze a website if it is idle for a while. Pauzing a dotnet process is the same as shutting
```
Comment (vadi2) [inline vonk/deployment/reverseProxy/iis.rst]:
```suggestion
   IIS will pause a website if it is idle for a while. Pausing a dotnet process is the same as shutting
```

Comment (mbaltus) [reply vonk/deployment/reverseProxy/iis.rst]: Yes, the images should render when they're deployed to RTD, but I'll keep an eye out for that. I'll make the change you suggested, and will then merge.

**Comment by vadi2** [comment]
It still says `Pauzing`

**Comment by mbaltus** [comment]
Oh! I searched for pauze instead of pauz, and didn’t notice the second one.
Corrected, so it should show up in a few minutes.

Thanks!

Mirjam

On 11 Nov 2019, at 13:30, Vadim Peretokin <notifications@github.com<mailto:notifications@github.com>> wrote:


It still says Pauzing

**Pull request merged**